### DISCUSSION TITLE
SFTP & 1Password

### DISCUSSION THREAD

**Discussion by Robert-M-Muench** [mikekazakov/nimble-commander#126] [General]
Is there any way how I can add an SFTP connection that uses the 1Password SHH agent? I have to enter a password or select a key file.

However, NC should follow the standard login methods configured in the OS, which it doesn't seem to do.

**Comment by mikekazakov** [comment]
Right now it's not supported.
However the library NC uses under the hood can use SSH agents, so it's possible to add this functionality.
PRs are welcome :)

**Reply by Robert-M-Muench** [reply]
That would be great... and yes, I know PRs are welcome, and I would like to do it, but I don't see a chance to get deep enough into the project.

**Reply by criticaldistance** [reply]
I second this feature as I also use 1Password to manage SSH keys.  Unfortunately I'm not a programmer so I wouldn't even know where to begin to attempt to implement this for the community :(

**Reply by mikekazakov** [reply]
@criticaldistance If you'd like to get involved there's plenty of ways beyond writing code, the [contributing guide](https://github.com/mikekazakov/nimble-commander/blob/main/CONTRIBUTING.md) is a good place to start.

**Reply by criticaldistance** [reply]
@mikekazakov happy to help in any way that I can. Thanks for linking me to that guide. I don't want to get off topic, but I have a couple other questions about features I've ran across that seem inconsistent, what would be the best way to inquire about those? Create a discussion thread or an issue?

**Reply by mikekazakov** [reply]
A discussion should be perfectly fine.
